plaidml2/bridge/keras/__init__.py

```diff
@@ -348,27 +437,34 @@ def categorical_crossentropy(target, output, from_logits=False):
     return _KerasNode('categorical_crossentropy', tensor=R)
 
 
+@_log_call
 def ceil(x):
-    _report_unimplemented('ceil')
+    return _KerasNode('ceil', tensor=edsl.ceil(x.tensor))
 
 
+@_log_call
 def clear_session():
-    _report_unimplemented('clear_session')
+    global _in_train_phase, _device
+    _in_train_phase = None
+    _dev = None
```

Comment:
I see `_device` and `_dev` here, that doesn't seem right

Reply:
Thanks!

Reply:
Whoops, my fix broke things, will investigate and put in a valid fix...

Reply:
✅